apps/src/sites/studio/pages/pd/professional_learning_landing/index.js

```diff
@@ -1,16 +1,21 @@
 import LandingPage from '@cdo/apps/code-studio/pd/professional_learning_landing/LandingPage';
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import getScriptData from '@cdo/apps/util/getScriptData';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+$(() => {
```

Comment:
not sure if this is actually necessary, but I wanted to wait for the page to load so that `analyticsReporter` was loaded. Definitely open for discussion/pushback if folks think this change isn't necessary!

Reply:
That makes sense! Would you mind adding a comment there to explain that its presence is for the Amplitude event please?